### PR TITLE
Use list as package name source

### PR DESCRIPTION
**Pull request by adipriyantobpn**

### CODE REVIEW

File: meta/.galaxy_install_info [hunk 1]
```diff
@@ -0,0 +1 @@
+{install_date: 'Tue Jul  4 10:56:19 2017', version: 2.0.0}
```
Comment (geerlingguy) [inline meta/.galaxy_install_info]: This file should not be added; it's added by galaxy upon role installation.

File: vars/default.yml [hunk 2]
```diff
@@ -1,2 +1,5 @@
 ---
-certbot_package: certbot
+certbot_package:
```
Comment (geerlingguy) [inline vars/default.yml]: Are `certbot-apache` and `certbot-nginx` required for certbot functionality?

I'd rather leave it up to the end user to install extra addons if they want...

File: meta/.galaxy_install_info (hunk 1 above)
Comment (adipriyantobpn) [inline meta/.galaxy_install_info]: Hi Jeff, thank you so much for reviewing.
It's my bad that incidentally adding this file. I'll remove it in other commit.
Maybe i could add meta/.galaxy_install_info into .gitignore.

File: vars/default.yml (hunk 2 above)
Comment (adipriyantobpn) [inline vars/default.yml]: Maybe certbot_package_extras variable can be added to install additional packages?
I'll add it in my other commit.

Comment (geerlingguy) [inline vars/default.yml]: Yeah, maybe we could switch it to `certbot_packages` for the first variable (a list with one item), and also have `certbot_packages_extra` for additional packages (and not run that task if none are defined).

That's a pattern I use in my PHP role here, and it works great: https://github.com/geerlingguy/ansible-role-php#role-variables

**Pull request closed without merging**